ci-operator/config/openshift-eng/rebasebot/openshift-eng-rebasebot-main.yaml

```diff
@@ -492,6 +492,68 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
+- as: tempo-operator
+  cron: 0 23 * * 1-5
+  steps:
+    test:
+    - as: tempo-operator
+      commands: |
+        rebasebot --source https://github.com/grafana/tempo-operator:main \
+                  --dest openshift/grafana-tempo-operator:main \
+                  --rebase IshwarKanse/tempo-operator:rebase-bot-main \
+                  --update-go-modules \
+                  --github-app-key /secrets/rebasebot/cloud-team-openshift-bot-key \
+                  --github-cloner-key /secrets/dt-secrets/dt-cloner \
+                  --github-cloner-id 811756 \
+                  --git-username cloud-team-rebase-bot \
+                  --git-email cloud-team-rebase-bot@redhat.com \
```

Comment:
Could you please change the references to the cloud-team, to avoid confusion on the ownership of this config?